3Dmol/glviewer.js

```diff
@@ -1202,6 +1206,32 @@ $3Dmol.GLViewer = (function() {
             ret.normalize();
             return ret;
         };
+        var spinInterval;
+        this.spin = function(spin,axis,interval){
```

Comment:
I was thinking of overloading the first argument to serve the purpose of both spin and axis.
```
spin() 
spin(true)
spin('y')
```
would all do the same thing
`spin(false)` deactivates